server/src/test/java/org/elasticsearch/index/mapper/IdLoaderTsidHashingTests.java

```diff
@@ -0,0 +1,275 @@
+/*
```

Comment:
Instead of duplicating this test. Maybe change `IdLoaderTests` to hash the tsid and add one or two tests that do use legacy tsid?